myapp/models.py

```diff
@@ -1,3 +1,22 @@
 from django.db import models
+import datetime
+import django.utils.timezone 
 
-# Create your models here.
+class Author(models.Model):
+    name = models.CharField(max_length = 30)
+    brithdate = models.DateField()
+    about_author = models.TextField()
+
+    def __str__(self):
+        return self.name
+
+class Blog(models.Model):
+    author = models.ForeignKey(Author, on_delete = models.CASCADE) 
+    title = models.CharField(max_length=70)
+    pub_date = models.DateTimeField(default=django.utils.timezone.now())
```

Comment:
Create a BaseModel that contains the `created_at` and `updated_at` fields. Inherit this model in each model you creates. So that you don't have to write created and updated time every time.